lib/stack_master/cli.rb

```diff
@@ -270,6 +271,13 @@ def execute_stacks_command(command, args, options)
       @kernel.exit false unless success
     end
 
+    def show_other_region_candidates(stack_name)
+      candidates = config.filter(stack_name=stack_name)
+      return if candidates.empty?
+
+      StackMaster.stdout.puts "Stack name #{stack_name} exists in regions: #{candidates.map(:&region).join(', ')}"
```

Comment:
I wonder if this should be more clearly worded as an error case, with a helpful suggestion.  e.g.:

```suggestion
      StackMaster.stdout.puts "Stack name #{stack_name} not found in region #{region}, but was found in regions: #{candidates.map(:&region).join(', ')}"
```

Reply:
This is already a part of the previous line: https://github.com/envato/stack_master/pull/343/files#diff-f83645dd9bf35512a532426db4fed419R256

Reply:
Oh -- so it does.